src/basic/io-util.h

```diff
@@ -106,10 +106,12 @@ static inline int iovw_consume(struct iovec_wrapper *iovw, void *data, size_t le
                 free(data);
         return r;
 }
+int iovw_put_dup(struct iovec_wrapper *iovw, const void *data, size_t len);
```

Comment:
why is this added in this commit? doesn't seem used

Reply:
Ouch, I forgot to drop it.